bootstrap/interfaces/secret.go

```diff
@@ -28,4 +28,10 @@ type SecretProvider interface {
 
 	// HasSecret returns true if the service's SecretStore contains a secret at the specified path.
 	HasSecret(path string) (bool, error)
+
+	// RegisteredSecretUpdateCallback registers a callback for a secret.
+	RegisteredSecretUpdateCallback(path string, callback func(path string))
```

Comment:
```suggestion
	// RegisteredSecretUpdatedCallback registers a callback for a secret.
	RegisteredSecretUpdatedCallback(path string, callback func(path string))
```

Reply:
Actually, "RegisterSecretUpdatedCallback" not "Register**ED**SecretUpdateCallback"